src/sass/static/pages.scss

```diff
@@ -973,28 +970,13 @@ body #not_authenticated_financial ul.checked > li {
             color: $COLOR_BLACK;
             border-radius: 0;
         }
-        .onfido-sdk-ui-Theme-textButton:hover {
-            color: $COLOR_BLACK;
-        }
-        .onfido-sdk-ui-CustomFileInput-container {
```

Comment:
you can comment this with some TODO message

Reply:
Done 😃